.github/workflows/linting.yml

```diff
@@ -535,7 +535,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: changed_files
     if: needs.changed_files.outputs.phanstubs == 'true' && github.event.pull_request.user.login != 'matticbot'
-    timeout-minutes: 5  # 2025-07-28: Probably takes about a minute.
+    timeout-minutes: 5  # 2025-11-06: Probably takes about a minute, but shouldn't run often.
```

Comment:
The "but shouldn't run often" isn't really all that relevant. 🤷 The point is to keep a hung job from hanging around for a very long time.